symfony/stimulus-bundle/2.19/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+    "bundles": {
+        "Symfony\\UX\\StimulusBundle\\StimulusBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "assets/": "assets/"
+    },
+    "aliases": ["stimulus", "stimulus-bundle"],
+    "conflict": {
+        "symfony/framework-bundle": "<7.2",
+        "symfony/security-csrf": "<7.2",
```

Comment:
wouldn't this prevent applying the whole stimulus-bundle recipe when installing StimulusBundle 2.19 with FrameworkBundle 6.4 ?

Reply:
flex skips conflicting recipes by version, so it'll install the version before without any pb